### PR TITLE
[CELEBORN-1566] Update docs about using HDFS

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### What changes were proposed in this pull request?
update client deployment doc to include the param (spark.celeborn.storage.activeTypes)
### Why are the changes needed?

Just provide a hint for users, otherwise they may miss this param.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Yes

### CODE REVIEW

**Comment by zhaohehuhu** [comment]
plz review this PR. Thanks @FMX

**Comment by FMX** [comment]
It looks like the UT failures are not relevant to this PR.  The README file should be updated too.

**Comment by zhaohehuhu** [comment]
> It looks like the UT failures are not relevant to this PR. The README file should be updated too.

Done. Thanks.

File: README.md [hunk 1]
```diff
@@ -291,6 +291,7 @@ spark.celeborn.client.push.replicate.enabled true
 spark.sql.adaptive.localShuffleReader.enabled false
 
 # If Celeborn is using HDFS
+spark.celeborn.storage.activeTypes HDFS
```
Comment (RexXiong) [inline README.md:294]: use spark.celeborn.storage.availableTypes instead.

**Review by RexXiong — APPROVED** [review]
LGTM

**Comment by RexXiong** [comment]
seems duplicated with https://github.com/apache/celeborn/pull/2675

**Comment by zhaohehuhu** [comment]
> seems duplicated with #2675, If you don't mind, we may need to close this PR.

OK. It's ok for me.

**Comment by RexXiong** [comment]
> This is not duplicated with #2675 . Updating different part of the docs , and for different purposes.

Thank you for clarifying, you are correct.

**Comment by RexXiong** [comment]
merge to main(v0.6.0)

**Pull request closed without merging**